plugins/knife/knife-box-list.rb

```diff
@@ -49,7 +49,8 @@ def run()
       name_glob = name_args.first
       name_glob = "*" if name_glob == "" or name_glob.nil?
       puts "Searching for node #{name_glob} in #{env}..."
-      puts "No such node" if find_nodes(name: name_glob, roles: "chef-base").each do |node|
+      nodes = find_nodes(name: name_glob, roles: "chef-base") + find_nodes(name: name_glob, roles: "cw_aio")
```

Comment:
I'd have to factor in the 'name' and 'chef-environment' stuff which I think just makes it too verbose.